assets/coffeescript/pupil_store.coffee

```diff
@@ -640,15 +641,127 @@ class PupilStore
         event.preventDefault()
         link = $(event.target)
         data = @_getOrderPermalink()
-        document.location = "?"+ $.param(data)
+        document.location = "?" + $.param(data) + '&user=uk' + '&test=1'
 
-  eventFillCartFromQueryString: ->
+  eventFillORderFormFromQueryString: ->
     query = window.location.search.substring(1)
+    urlParams = new URLSearchParams(window.location.search)
+    testQuery = urlParams.get('test')
+    userQuery = urlParams.get('user')
+
+    name = {
+      us : 'John',
+      uk : 'George'
+      de : 'Christen',
+      jp : 'Takeshi',
+    }
+
+    contact_info = {
+      contact: {
+        email_c : 'test@example.com',
+        name_0_c : name[userQuery] + 'C',
+        name_1_c : name[userQuery] + 'C',
```

Comment:
lastname should be different than first name (if you want to do something simple just do `name[userQuery] + 'surnameC'`